File: spring/jwt-token-auth-demo/src/main/java/com/sggw/jwttokenauthdemo/service/HealthCheckService.java
package com.sggw.jwttokenauthdemo.service;

import org.springframework.core.env.Environment;
import org.springframework.stereotype.Service;

import lombok.RequiredArgsConstructor;

@RequiredArgsConstructor
@Service
public class HealthCheckService {
    
    private final Environment environment;

    public String healthCheck() {
        return environment.getProperty("profile.message") + " is live";
    }
}

File: nestjs/jwt-token-auth/src/health-check/health-check.service.ts
import { Injectable } from '@nestjs/common';
import { ConfigService } from '@nestjs/config';

@Injectable()
export class HealthCheckService {

    constructor(private configService: ConfigService) {}

    healthCheck() {
        return `${this.configService.get<string>("profile.message")} is live`;
    }
}
